### DISCUSSION TITLE
Github shows 1 fork, I must have messed up, how do I fix?

### DISCUSSION THREAD

**Discussion by cricri-pingouin** [community/community#23769] [New to GitHub]
<p>Hi,</p>
<p>I joined and uploaded a bunch of my stuff yesterday, but one of my repo shows 1 fork even though it doesn’t have any:</p>
<p><a href="https://github.com/cricri-pingouin/SliMind" rel="nofollow noopener">https://github.com/cricri-pingouin/SliMind</a></p>
<p>I must have messed up somehow, but don’t know how to fix it!</p>
<p>Anyone can help?</p>
<p>Thanks!</p>

**Comment by yamiacat** [comment]
<p>Hi there! :wave: I don’t see that there are any forks that have been made from that repository?</p>
<p>The branch icon and fork icon look very similar, and your repository has 1 branch - could that be what you saw?</p>

**Comment by yamiacat** [comment]
<p>Oh, I see you deleted and recreated the repository, that’s why I can’t see a fork!</p>
<p>In that case, I think I know what happened. Sometimes some of our users are reported for spammy behavior, and if that happens their profile gets hidden from everyone. If they have forked a repository, that still gets counted however.</p>
<p>So if you saw that a fork had been made, but there was no username visible to say who forked it, they had probably been reported!</p>
<p>I hope that clears things up!</p>

**Comment by cricri-pingouin** [comment, marked as answer]
<p>Ah, ok, thanks!<br>
Yes, I fixed a bug at some point, so I just deleted the repository and recreated it while uploading the fix, killing 2 birds with 1 stone!</p>
